src/sentry/utils/audit.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+from sentry.models import ApiKey, AuditLogEntry
+
+import logging
+audit_logger = logging.getLogger('sentry.audit.api')
+
+
+def create_audit_entry(request, transaction_id=None, **kwargs):
+
+    user = request.user if request.user.is_authenticated() else None
+    api_key = request.auth if hasattr(request, 'auth') \
+        and isinstance(request.auth, ApiKey) else None
+
+    entry = AuditLogEntry(
+        actor=user, actor_key=api_key, ip_address=request.META['REMOTE_ADDR'], **kwargs
+    )
+
+    # Only create a real AuditLogEntry record if we are passing an event type
+    # otherwise, we want to still log to our actual logging
+    if entry.event is not None:
+        entry.save()
+
+    extra = {
+        'ip_address': entry.ip_address,
+        'organization_id': entry.organization_id,
+        'object_id': entry.target_object,
+        'entry_id': entry.id,
+        'actor_label': entry.actor_label
+    }
+    if entry.actor_id:
+        extra['actor_id'] = entry.actor_id
+    if entry.actor_key_id:
+        extra['actor_key_id'] = entry.actor_key_id
+    if transaction_id is not None:
+        extra['transaction_id'] = transaction_id
+
+    audit_logger.info(entry.get_event_display(), extra=extra)
```

Comment:
So there is one change in behavior here. The audit logger is different, because we use a different logger for the API vs the UI. One is `sentry.audit.api` and the other is `sentry.audit.ui`.

Given this, I'd say we can pass the logger in as an argument into `create_audit_entry`. That way each caller of this has an opportunity to define their own logger to use.

Let me know if this makes sense or need more clarification.